### PR TITLE
feat: added Swedish translation

### PR DESCRIPTION
**Pull request by Joakim-T**
## Pull Request Checklist

- [X] **Description:** Briefly describe the changes in this pull request.
- [X] **Changelog:** Ensure a changelog entry following the format of [Keep a Changelog](https://keepachangelog.com/) is added at the bottom of the PR description.
- [ ] **Documentation:** Have you updated relevant documentation [Open WebUI Docs](https://github.com/open-webui/docs), or other documentation sources?
- [ ] **Dependencies:** Are there any new dependencies? Have you updated the dependency versions in the documentation?
- [X] **Testing:** Have you written and run sufficient tests for the changes?
- [X] **Code Review:** Have you self-reviewed your code and addressed any coding standard issues?

---

## Description

Added Swedish translation

---

### Added

- Folder sv-SE in /src/lib/i18n/locales/
- File translation.json with Swedish translation based on en-US translation.json

### Changed

Added sv-SE to /src/lib/i18n/locales/languages.json

### Breaking Changes

---

### CODE REVIEW

File: src/lib/i18n/locales/languages.json [hunk 1]
```diff
@@ -95,4 +95,8 @@
 		"code": "zh-TW",
 		"title": "Chinese (Traditional)"
 	}
+	{
```
Comment (justinh-rahb) [inline src/lib/i18n/locales/languages.json:98]: This list should be sorted alphabetically according to the code (except is the default en-US at the top).

**Review by Joakim-T — COMMENTED** [review]
Moved sv-SE to correct position in languages.json

**Comment by justinh-rahb** [comment]
If I can make one more request: a more descriptive title for the PR and I'd say we're set 👍

**Comment by tjbck** [comment]
Thanks!

**Pull request merged**